build/Jenkinsfile.branch

```diff
@@ -18,7 +18,7 @@ pipeline {
   }
 
   options {
-    buildDiscarder(logRotator(numToKeepStr: '3'))
+    buildDiscarder(logRotator(numToKeepStr: '30'))
```

Comment:
What is the reasoning behind only keeping a certain number of build vs. keeping them for a certain number of day?

Reply:
no reason, maybe to keep easier the disk space on jenkins :)

but we can change for daysToKeepStr and put lets set 10 days

Reply:
Yes, let's do that instead and set the number to 15. We can decrease it if we see there is pressure on the disk space.